Gems/WhiteBox/Code/Source/Viewport/WhiteBoxPolygonTranslationModifier.cpp

```diff
@@ -280,6 +280,7 @@ namespace WhiteBox
 
         Api::VertexPositionsCollection outlines = Api::PolygonBorderVertexPositions(*whiteBox, m_polygonHandle);
         AZStd::vector<AZ::Vector3> triangles = Api::PolygonFacesPositions(*whiteBox, m_polygonHandle);
+        AZ::Vector3 normal = Api::PolygonNormal(*whiteBox, m_polygonHandle);
```

Comment:
nit: This should be `const` and we might want to move this closer to the first use (nearer line 294)